src/App.tsx

```diff
@@ -14,11 +15,112 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+enum SortBy {
+  name,
+  length,
+  null,
+}
+
+const App: React.FC = () => {
+  let goodsList = [...goodsFromServer];
+  const [isListVisible, setStart] = useState(false);
+  const [isReverse, setReverse] = useState(false);
+  const [sortBy, setSortBy] = useState(SortBy.null);
+
+  const start = () => {
+    setStart(prev => !prev);
```

Comment:
in case of `reverse` inverting (`!`) makes total sense,
but when start used only once and then the button is not visible,
so inverting does not fit, regular `true` would be just fine
yet it does not impact much